.github/workflows/merge-gatekeeper.yml

```diff
@@ -0,0 +1,27 @@
+name: Merge Gatekeeper
+
+on:
+  pull_request:
+    branches:
+      - main
+      - dev
```

Comment:
We don't have a `dev` branch, so I guess it's not needed to add it.

Reply:
I change it to `develop` since we do make breaking change PR to `develop` from time to time.